src/components/Feedback.tsx

```diff
@@ -36,11 +36,22 @@ interface Props {
 }
 
 export const Feedback: React.FC<Props> = ({ taxFormUserInput }: Props) => {
-  const [isOpen, setIsOpen] = useState(false)
+  const [isFeedbackOpen, setIsFeedbackOpen] = useState(false)
+  const [isUsefulSubmitted, setIsUsefulSubmitted] = useState(false)
   const [isSubmittedSuccessfully, setIsSubmittedSuccessfully] = useState(
     undefined,
   )
-  if (isOpen) {
+
+  const usefulOnClick = () => {
+    /** TODO make actually do something */
```

Comment:
Bez tohto je nam to nanic :)

Reply:
Hlavny zmysel tohto PR je dat prec tie priblble badge a zjednotit to s navodmi

Tu url som zobral z navodov a pytal som sa vtedy davno Jana ci mozeme pouzit ich api, ale nedostal som odpoved. 

Tak bud mozeme:
1) Zatial to nechat tak, je to nice to have
2) Riesit vlastne API a ukladat si to
3) Riesit integraciu na navody

Zatial asi hlasujem za 1 :)

Reply:
Ok, suhlasim. V tom pripade by som skryl tu otazku ci je to uzitocne, kedze to nema ziadnu funkcionalitu a nechame ten priestor prazdny. Ak to tam bude Hana chciet tak vysvetlime ze to nie je jednoduche a moze to tam byt v dalsej verzii (plus vysvetlim ze ta otazka nema zmysel pri appke ako som pisal v druhom komente).

PS: Nezabudni vyriesit konflikty 😉

Reply:
No neviem ci to nebude vyzerat zvlastne prazdne, ale bolo by aj zlvastne tam mat nieco co nefunguje :D Iba ak tak pre dobry pocit usera ze dal feedback :D

Ake konflikty? Ja ziadne nevidim.

Reply:
Mne Github pise
![Screenshot 2020-06-24 at 20 52 14](https://user-images.githubusercontent.com/1561771/85615495-9a49dd80-b65c-11ea-87d8-8ec54f8b55e2.png)